oci/layout/oci_transport.go

```diff
@@ -85,41 +84,44 @@ type ociReference struct {
 	// (But in general, we make no attempt to be completely safe against concurrent hostile filesystem modifications.)
 	dir         string // As specified by the user. May be relative, contain symlinks, etc.
 	resolvedDir string // Absolute path with no symlinks, at least at the time of its creation. Primarily used for policy namespaces.
-	tag         string
+	image       string
 }
 
 // ParseReference converts a string, which should not start with the ImageTransport.Name prefix, into an OCI ImageReference.
 func ParseReference(reference string) (types.ImageReference, error) {
-	var dir, tag string
-	sep := strings.LastIndex(reference, ":")
-	if sep == -1 {
-		dir = reference
-		tag = "latest"
-	} else {
-		dir = reference[:sep]
-		tag = reference[sep+1:]
+	var dir, image string
+	sep := strings.SplitN(reference, ":", 2)
+	dir = sep[0]
+
+	if len(sep) == 2 {
+		if strings.Contains(sep[1], ":") {
+			image = sep[1]
+		} else {
+			image = sep[1] + ":latest"
+		}
```

Comment:
@mtrmac I get it now. The reason I was adding `latest` to the image name if a tag was not given was because of the way kpod and docker do things. For example, if a user downloads the `alpine:latest` image from a registry and decides to save to oci on disk using the command `kpod save --oci -o alp.tar alpine`, the image name that will be saved in index.json based on your comment will be `alpine`. 

Therefore, when `alp.tar` is loaded back into containers-storage using the `kpod load` command, it will be stored as `alpine` instead of `alpine:latest`. I feel like this alters some form of the identity of the image as there would be no tag associated with it.

If we want to give the user the freedom of setting the image name to whatever they want it to be, we need to make it very clear in the documentation since all the kpod commands allow the user to access images with `latest` tag by just passing in the name.